oneshot/backfill_abuser_data.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+import argparse
+from typing import List
+
+from app.abuser_utils import check_if_abuser_email, mark_user_as_abuser
+from app.db import Session
+from app.models import User
+
+parser = argparse.ArgumentParser(
+    prog="Backfill abuser data",
+    description="Iterate over disabled users and generate abuser bundle",
+)
+args = parser.parse_args()
+
+print("I'm going to generate abuser bundle for disabled users.")
+
+disabled_users: List[User] = Session.query(User).filter(User.disabled == True).all()  # noqa: E712
```

Comment:
We need to do this by batches with userId > n and userId < m because we don't have an index on disabled and this will trigger a full table scan